rdkit/Dbase/DbConnection.py

```diff
@@ -182,12 +171,10 @@ def GetData(self, table=None, fields='*', where='', removeDups=-1, join='', tran
       **Notes**
 
         - this uses _DbUtils.GetData_
-       
 
     """
-    if table is None:
-      table = self.tableName
-    kwargs['forceList'] = kwargs.get('forceList', 0)
+    table = table or self.tableName
+    kwargs.setdefault('forceList', 0)
```

Comment:
I'm curious why you consider this formulation better than what was there already.
It seems like the `setdefault` method has very different semantics (not that it matters in this case)